invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -456,6 +456,7 @@ class CommunityProfileForm extends Component {
                         community={community}
                       >
                         <RemoteSelectField
+                          additionLabel={i18next.t("Add organization: ")}
```

Comment:
Isn't this going to change the UI for everyone? At a minimum we need a screenshot of how this looks.